containers/vpn/OpenVPNAccountSection/OpenVPNCredentialsModal.js

```diff
@@ -25,6 +25,13 @@ const OpenVPNCredentialsModal = ({ username = '', password = '', fetchUserVPN, .
     const handleChangePassword = ({ target }) => setCredentials({ ...credentials, password: target.value });
 
     const handleSubmit = async () => {
+        const RESERVED_WORDS = ['guest'];
```

Comment:
What about moving `RESERVED_WORDS` outside of the `handleSubmit` function?

Reply:
I thought about moving it outside but wanted to keep it close to the context it's used in (as it's not being reused anywhere else). RESERVED_USERNAMES definitely sounds better :)